### PR TITLE
Radio styles

### PR DESCRIPTION
**Pull request by sineade**
:art:
Added in some styling to make centering the radio buttons to be done in bem, and applying styles to the correct places.

### CODE REVIEW

File: app/assets/stylesheets/ama_layout/layout_components/comparison-radios.scss [hunk 1]
```diff
@@ -34,21 +34,36 @@
...
 
-  &__radio-btn--centered{
-    padding: 0 0 21px 0;
-
-    &:before{
-      top: 100%;
-      margin-top: -35px;
-      left: 50%;
-      margin-left: -12px;
+  &__radio-btn--centered {
```
Comment (kathleenAMA) [inline app/assets/stylesheets/ama_layout/layout_components/comparison-radios.scss]: Shouldnt be a space before {

File: app/assets/stylesheets/ama_layout/layout_components/comparison-radios.scss [hunk 2]
```diff
@@ -34,21 +34,36 @@
...
+
+    input[type="radio"] + label{
+      padding: 0 0 46px 0;
+
+      &:before{
```
Comment (kathleenAMA) [inline app/assets/stylesheets/ama_layout/layout_components/comparison-radios.scss]: the single : is from CSS2 so I think we're trying to move over to :: which is CSS3.

File: app/assets/stylesheets/ama_layout/layout_components/comparison-radios.scss [hunk 3]
```diff
@@ -34,21 +34,36 @@
...
+        top: 100%;
+        margin-top: -35px;
+        left: 50%;
+        margin-left: -12px;
+      }
+
+      &:after{
```
Comment (kathleenAMA) [inline app/assets/stylesheets/ama_layout/layout_components/comparison-radios.scss]: same here with the double ::

File: app/assets/stylesheets/ama_layout/layout_components/comparison-radios.scss [hunk 4]
```diff
@@ -34,21 +34,36 @@
...
 
-    &:after{
-      top: 100%;
-      margin-top: -27px;
-      left: 50%;
-      margin-left: -4px;
+    input[type="radio"]:checked + label{
+      padding: 0 0 46px 0;
+
+      &:after{
```
Comment (kathleenAMA) [inline app/assets/stylesheets/ama_layout/layout_components/comparison-radios.scss]: and here ::

**Comment by kathleenAMA** [comment]
Also needs master :hankey:

**Comment by sineade** [comment]
@kathleenAMA they are only in forms2 at the moment. But should be able to use them on join as well.

**Comment by kathleenAMA** [comment]
:art: :art: :art:

**Pull request merged**